client/deis.py

```diff
@@ -1321,24 +1322,34 @@ def config_push(self, args):
         """
         Sets environment variables for an application.
 
-        Your environment is read from a file named .env. This file can be
-        read by foreman to load the local environment for your app.
+        Your environment is read from the optionally specified path to a
+        .env-formatted file. If no path is specified, your environment is read
+        from a file named .env. This file can be read by foreman to load the
+        local environment for your app.
 
-        Usage: deis config:push [options]
+        Usage: deis config:push [<path>] [options]
+
+        Arguments:
+          <path>
+            a path leading to an environment file
```

Comment:
using the [Options Descriptions Format](https://github.com/docopt/docopt#option-descriptions-format) in docopt, let's set up `.env` as the default so we don't have to spell it in the code. Also makes he docs a bit clearer with the comment about updating the docs